.cirrus.yml

```diff
@@ -42,5 +48,32 @@ release_task:
     - python3.7 -m ensurepip
     - python3.8 -m ensurepip
   build_script: ./make-nix-release.sh
+  script: |
+    #!/usr/bin/env bash
+
+    if [[ "$CIRRUS_RELEASE" == "" ]]; then
```

Comment:
I've discovered that jobs won't fire with `$CIRRUS_RELEASE` unless the release is tagged from the web UI. I'd suggest just eliding this check since we check for `$CIRRUS_TAG` above.